plugins/importexport/native/NativeImportDom.inc.php

```diff
@@ -813,6 +813,10 @@ function handleArticleNode(&$journal, &$articleNode, &$issue, &$section, &$artic
 		}
 		if ($hasErrors) return false;
 		
+		$citationDao =& DAORegistry::getDAO('CitationDAO');
```

Comment:
Hi Alec,
Indeed, I have tested it twice. I have just imported some old issue XML (in
attachment) to  http://21bs.ru/index.php/bio/issue/view/24/showToc with no
problem.
Warning: citations text must not contain html entities or symbols: no &, no
&amp; etc.

Then, would it be useful for my pull request to check it with PHP 5.3?
Thank You.





*С уважением,Игорь Перескоков,*
*администратор сайтов,*
*ТПГ "ПетроСити"*

*м.т. +7-931-288-16-79*
| www.petrocity.ru | nskenergo.com | 21mm.ru | 21bs.ru | mmps.ru |
pramac-spb.ru | teplocity.spb.ru | poltavamash.su

2016-05-27 15:53 GMT+03:00 Alec Smecher <notifications@github.com>:

> In plugins/importexport/native/NativeImportDom.inc.php
> <https://github.com/pkp/ojs/pull/853#discussion_r64899653>:
>
> > @@ -813,6 +813,10 @@ function handleArticleNode(&$journal, &$articleNode, &$issue, &$section, &$artic
> >  		}
> >  		if ($hasErrors) return false;
> >  		
> > +		$citationDao =& DAORegistry::getDAO('CitationDAO');
>
> Have you tested an import? The tool we use for continuous integration
> tests fails on the import test, and it's probably because of this section,
> I think.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pkp/ojs/pull/853/files/44fda891526fa8003f7cb213b095c61499117987#r64899653>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASsrb_QOM72nZGWjPUs3PlxbhSXwPE8pks5qFulMgaJpZM4IoY9E>
> .
>

OJS Version
Current version
2.4.8.0 (March 18, 2016 - 12:23 PM)

Check for updates

Version history
 
VERSION	MAJOR	MINOR	REVISION	BUILD	DATE INSTALLED
 
2.4.8.0	2	4	8	0	2016-03-18
 
2.4.7.1	2	4	7	1	2016-01-19
 
2.4.6.0	2	4	6	0	2015-07-05
 
2.4.2.0	2	4	2	0	2013-08-06
 
OJS Configuration
OJS configuration settings from config.inc.php.

general
installed	On
base_url	http://21bs.ru
registry_dir	registry
session_cookie_name	OJSSID
session_lifetime	30
scheduled_tasks	Off
scheduled_tasks_report_error_only	On
date_format_trunc	%m-%d
date_format_short	%Y-%m-%d
date_format_long	%B %e, %Y
datetime_format_short	%Y-%m-%d %I:%M %p
datetime_format_long	%B %e, %Y - %I:%M %p
time_format	%I:%M %p
disable_path_info	Off
allow_url_fopen	Off
restful_urls	Off
trust_x_forwarded_for	Off
enable_cdn	On
citation_checking_max_processes	3
show_upgrade_warning	On
enable_beacon	On
database
driver	mysql
host	localhost
username	
password	
name	
persistent	Off
debug	Off
cache
object_cache	none
memcache_hostname	localhost
memcache_port	11211
web_cache	Off
web_cache_hours	1
i18n
locale	en_US
client_charset	utf-8
connection_charset	utf8
database_charset	utf8
charset_normalization	utf8
files
files_dir	files
public_files_dir	public
umask	18
finfo
security
force_ssl	Off
force_login_ssl	Off
session_check_ip	On
encryption	md5
salt	YouMustSetASecretKeyHere!!
reset_seconds	7200
allowed_html	<a href|target> <em> <strong> <cite> <code> <ul> <ol> <li> <dl> <dt> <dd> <b> <i> <u> <img src|alt> <sup> <sub> <br> <p>
email
enable_attachments	On
time_between_emails	3600
max_recipients	10
require_validation	Off
validation_timeout	14
search
min_word_length	3
results_per_keyword	500
result_cache_hours	1
oai
oai	On
repository_id	ojs21bs.ru
oai_max_records	100
interface
items_per_page	25
page_links	10
captcha
captcha	On
captcha_on_register	On
captcha_on_comments	On
captcha_on_mailinglist	On
font_location	/files/browa.ttf
recaptcha	Off
cli
perl	/usr/bin/perl
tar	/bin/tar
egrep	/bin/egrep
gzip	/bin/gzip
xslt_command	
proxy
debug
show_stats	Off
show_stacktrace	Off
display_errors	Off
deprecation_warnings	Off
log_web_service_info	Off
lockss
Server Information
Basic operating system and server software versions. Click on Extended PHP Information to view extended details of this server's PHP configuration.

OS platform	Linux
PHP version	5.4.16
Apache version	Apache/2.4.6 (CentOS) OpenSSL/1.0.1e-fips mod_fcgid/2.3.9 PHP/5.4.16 mod_wsgi/3.4 Python/2.7.5
Database driver	mysql
Database server version	5.5.44-MariaDB
Extended PHP Information